travis/build-wheels.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e -x
+
+PYBIN=/opt/python/$PYVER/bin
+
+"${PYBIN}/pip" install --upgrade setuptools  # work around https://github.com/pypa/setuptools/issues/1086
+
+# Compile wheels
+"${PYBIN}/pip" install -r /io/requirements.txt
+
+# Install dependencies of asynq
+"${PYBIN}/pip" install -U Cython qcore tox
```

Comment:
We should just add these to requirements.txt instead (not sure why we didn't do that before).